src/test/java/jenkins/plugins/slack/workflow/SlackSendStepTest.java

```diff
@@ -51,6 +53,14 @@
     @Mock
     SlackNotifier.DescriptorImpl slackDescMock;
 
+    String savedResponse = "{\"ok\":true,\"channel\":\"F4KE1DABC\",\"ts\":\"1543931401.000500\"," +
```

Comment:
nit, to clean it up, could you put it in a json file and load it as a resource?
no need for string escaping then and would be much easier to read?